deployments/k8s/templates/deployment.yaml

```diff
@@ -40,6 +40,19 @@ spec:
             - name: http
               containerPort: {{ .Values.ports.http }}
               protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /
+              port: 80
+            initialDelaySeconds: 30
+            periodSeconds: 60
```

Comment:
I think 60 seconds is way too much. It means we won't be able to detect that the app is not able to serve traffic for, at least a minute.
An HTTP call is not that expensive for being that cautious here so I would either set the default (10 sec) or similar.

Anyway, what do you think about converting this to variables?

Reply:
@smoya Thankyou for review, I agree with you. We should reduce the periodSeconds time time in both liveness and readiness probes. I should have thought about it before, that 1 minute will be too much downtime for any container. I will set it to 10 sec for liveness and 6 seconds for readiness. Will it be Okay?
     Also It is good idea to create variables for both probes. I will create variable for all parameters (including path, initialDelaySeconds and periodSeconds ) of both the probes in values.yaml. Should it be Okay to create Variables  for all the parameters?

Reply:
>  I will set it to 10 sec for liveness and 6 seconds for readiness. Will it be Okay?

Yeah I think those are good values! 👍 

> I will create variable for all parameters (including path, initialDelaySeconds and periodSeconds ) of both the probes in values.yaml.

I don't think you need to convert things like the path into a variable since this won't change unless you change your application code explicitly. However, the others I suggested (`initialDelaySeconds` and `periodSeconds`) can vary depending on the resources given to the container. I.e. The same app won't perform equally in a 2 Core 2 GB RAM than in a 2 Core 4 GB RAM, so you might want to tune the values.

I hope it makes sense!

Reply:
Yes @smoya , I will make the suggested changes and commit it ASAP.